src/testing/jest/jest-27-and-under/jest-runner.ts

```diff
@@ -49,7 +50,7 @@ export async function runJest(config: d.ValidatedConfig, env: d.E2EProcessEnv) {
  * Creates a Stencil test runner
  * @returns the test runner
  */
-export function createTestRunner(): any {
```

Comment:
This was previously typed as `any` - `JestTestRunner` is just a type alias for `any` at this point, but I saved that piece of tech debt for another day - https://github.com/ionic-team/stencil/pull/4883/files#diff-4b0dc677b3f6e55c5f547f5585915852d874a4a1ff77a6211becc016251521fcR38-R39